src/components/look/look-list/style-tag-list/index.tsx

```diff
@@ -21,15 +21,15 @@ function StyleTagList({ styleTags, id }: StyleTagListProps) {
 
   const handleCheck = (id: number) => {
     const isSelected = selectedTagIds.includes(id);
-    setSelectedTagIds(selectedTagIds.concat(isSelected ? [] : id));
+    setSelectedTagIds(
+      selectedTagIds
+        .filter((selectedTagId) => selectedTagId != id)
+        .concat(isSelected ? [] : id)
+    );
   };
 
   const handleSave = async () => {
-    const newStyleTags = STYLE_TAG.filter((styleTag) =>
-      selectedTagIds.includes(styleTag.id)
-    );
-    console.log('newStyleTags', newStyleTags);
-    // await LookService.partialUpdate(id, { styleTags: newStyleTags });
+    await LookService.partialUpdate(id, { styleTagIds: selectedTagIds });
```

Comment:
```suggestion
    try {
      await LookService.partialUpdate(id, { styleTagIds: selectedTagIds });
      message.success('성공적으로 저장되었습니다.');
    } catch (error) {
      message.success('실패했습니다.' + error?.message); 
    }
```

서버 관련 요청할 때는 try-catch 활용해서 에러핸들링 해주는 것도 좋습니다!